AdyenActions/UI/View Controllers/QR Code/QRCodeViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal final class QRCodeViewController: UIViewController {
+    
+    private let qrCodeView: UIView
+    
+    internal init(qrCodeView: UIView) {
+        self.qrCodeView = qrCodeView
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    @available(*, unavailable)
+    internal required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    internal override func viewDidLoad() {
+        super.viewDidLoad()
+        buildUI()
+        view.backgroundColor = .clear
```

Comment:
you can set viewController's view as `qrCodeView` on 'loadView()', right ?

Reply:
yes, that's much better 👍